api/client/volume/create.go

```diff
@@ -26,16 +26,24 @@ func newCreateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "create",
+		Use:   "create [VOLUME]",
```

Comment:
Minor thing, you can add `[OPTIONS]` here to prevent it from being added at the end (for consistency):

```
create [OPTIONS] [VOLUME]
```